src/model/table.ts

```diff
@@ -1,12 +1,8 @@
 import { Column, ColumnInstance, Row } from 'react-table';
 
-export type ColumnDefinition<T extends object> =
-  Column<T>
-  & ColumnProperties<T>;
+export type ColumnDefinition<T extends object> = Column<T> & ColumnProperties<T>;
 
-export type ColumnInstanceDefinition<T extends object> =
-  ColumnInstance<T>
-  & ColumnProperties<T>;
+export type ColumnInstanceDefinition<T extends object> = ColumnInstance<T> & ColumnProperties<T>;
```

Comment:
Script `isready` probably messed up the formatting.
From ESLint: `This line has a length of 97. Maximum allowed is 80.(max-len)`
Please restore the correct formatting in this file and the others concerning the table.

Reply:
Fck, style changes happened with develop merge... I haven't noticed.
Overall, looking at those changes I find this styling better, what do you think?
BTW Current line width is set to 120, so this style probably will apply further.

Reply:
Overall I agree - I would suggest staying at 120 (max-len).